setup.cfg

```diff
@@ -5,10 +5,12 @@ max-line-length = 88
 [mypy]
 disallow_untyped_defs = True
 ignore_missing_imports = True
+follow_imports = silent
```

Comment:
Doesn't mypy allow a `files` option here? (Maybe it doesn't, but I think it does.) It would then be picked up by IDEs too. :)

Reply:
done